### DISCUSSION TITLE
Massive performance improvement. map will be shifted instead of redrawn

### DISCUSSION THREAD

**Discussion by Kellog** [mikes222/mapsforge_flutter#112] [General]
Oh... that sounds good! When do you think this will be working? Thanks!

**Comment by Kellog** [comment]
Will this also improve the zoom? Currently the redraw done on zoom finish is somewhat off position from the last pinched position.

**Comment by mikes222** [comment]
Thx for the bug report. I will investigate in the next days...

**Comment by mikes222** [comment]
This bug is solved.
